python_wrapper/susi_python/main.py

```diff
@@ -108,6 +108,8 @@ def generate_result(response):
         elif isinstance(action, StopAction):
             result['stop'] = action
             break
+        elif isinstance(action,MediaAction):
```

Comment:
Coding style

Reply:
@hongquan , what is wrong with the coding style?

Reply:
@stealthanthrax https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Didn't I tell you many times to add plugin to your code editor, to let it check automatically?